easybuild/easyconfigs/c/CompareGenotypeCalls/CompareGenotypeCalls-1.8.1-Java-8-LTS.eb

```diff
@@ -0,0 +1,29 @@
+name = 'CompareGenotypeCalls'
+version = '1.8.1'
+
+homepage = 'https://molgenis26.gcc.rug.nl/downloads/'
+description = """"""
+
+toolchain = {'name': 'system', 'version': ''}
+easyblock = 'Tarball'
+
+# download
+source_urls = ['https://molgenis26.gcc.rug.nl/downloads/']
```

Comment:
Dit kan, maar zou het niet netter zijn om de nieuwe DNS entry te gebruiken:
https://download.molgeniscloud.org/downloads/
of nog beter als de ```CompareGenotypeCalls-%(version)s-SNAPSHOT-dist.zip``` als extra artifact wordt toegevoegd bij de "release" op GitHub. Dan is het meteen duidelijk waar de code vandaan komt.